build_deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE_NAME="quay.io/cloudservices/compliance-backend"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+# Replace before merging
+#if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+if [[ ! ( -z "$QUAY_USER" || -z "$QUAY_TOKEN" ) ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
```

Comment:
This looks like a temporary directory. `mktemp -d` might be used here.

Reply:
mmm , not really sure we're going to get an advantage from using `mktemp` as it will create a randomly named directory (see https://ss64.com/bash/mktemp.html) but won't remove it or anything. If anything we'll get a new directory every time we'll have to remove manually.

Besides, this line comes straightly from [this example](https://internal.cloud.redhat.com/docs/devprod/tutorial/03-building/#integrating-into-appsres-jenkins) - I used the script there as a template to create our scripts, not really sure if we can use a different name scheme at all, so I'd say it's okay to keep the `mkdir -p` approach.